pkg/registry/apis/dashboard/search_test.go

```diff
@@ -0,0 +1,100 @@
+package dashboard
+
+import (
+	"context"
+	"fmt"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/grafana/grafana/pkg/apimachinery/identity"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/infra/tracing"
+	"github.com/grafana/grafana/pkg/services/user"
+	"github.com/grafana/grafana/pkg/storage/unified/resource"
+	"google.golang.org/grpc"
+)
+
+func TestSearchHandlerFields(t *testing.T) {
+	// Create a mock client
+	mockClient := &MockClient{}
+
+	// Initialize the search handler with the mock client
+	searchHandler := SearchHandler{
+		log:    log.New("test", "test"),
+		client: mockClient,
+		tracer: tracing.NewNoopTracerService(),
+	}
+
+	t.Run("Multiple comma separated fields will be appended to default dashboard search fields", func(t *testing.T) {
+		rr := httptest.NewRecorder()
+		req := httptest.NewRequest("GET", "/search?fields=field1,field2,field3", nil)
```

Comment:
all other array query params are arrays.  would be good to be consistent
```suggestion
		req := httptest.NewRequest("GET", "/search?field=field1&field=field2&field=field3", nil)
```

Reply:
good catch, didnt notice this!